<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgChooseAviator</class>
 <widget class="QDialog" name="DlgChooseAviator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgChooseAviator{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,  stop:0 	rgb(1, 18, 71), stop:0.5 rgb(0, 58, 120), stop:1 rgb(1, 18, 71));
}

#DlgChooseAviator QLabel{
	color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Aviator List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lvAviator"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ID Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbIDNumber">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;ID Number&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbName">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;Name&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
